app/controllers/orgs/lti_configurations_controller.rb

```diff
@@ -44,6 +47,39 @@ def destroy
       redirect_to edit_organization_path(id: current_organization), alert: "LTI configuration deleted."
     end
 
+    # rubocop:disable Metrics/MethodLength
+    def autoconfigure
+      builder = GitHubClassroom::LTI::ConfigurationBuilder.new("GitHub Classroom", auth_lti_launch_url)
+
+      builder.add_attributes(
+        description: "Sync your GitHub Classroom organization with your Learning Management System.",
+        icon: "https://classroom.github.com/favicon.ico",
+        vendor_name: "GitHub Classroom",
+        vendor_url: "https://classroom.github.com/"
+      )
+
+      ## LMS Specific Attributes ##
+      # Note: LMS's will ignore vendor identifiers they do not understand
+
+      # Canvas will not display an LTI application
+      # unless specify a course_navigation location.
+      builder.add_vendor_attributes(
```

Comment:
Do we think refactoring the meat of this entire method into a controller concern is worth consideration? It's already pretty long as it is, but I could easily imagine a future in which we have custom attributes for all of our most popular LMSs to ensure the best experience possible, and in that case this controller becomes a glorified config file 😬

Reply:
I think this is fine for now :+1: